src/lib/tool_providers.ml

```diff
@@ -494,7 +506,7 @@ let default_toolkit
     star_tool ~host ~meta_playground;
     stringtie_tool ~host ~meta_playground;
     cufflinks_tools ~host ~meta_playground;
-    hisat_tool ~host ~meta_playground;
+    hisat_tool ~host ~meta_playground ~version:`V_0_1_6_beta;
```

Comment:
Yea, actually was just thinking that - the two are not really linked. Here I create `V_0_1_6_beta` tool, and there I say if you ask for the default it's the V2 one (which may not be in the kit unless you add it)